go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/openshift/images
+
+go 1.21.6
```

Comment:
This should be `go 1.20`.

Reply:
Fixup in https://github.com/openshift/images/pull/168.